xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminUsersSheet.xml

```diff
@@ -244,7 +244,10 @@
 #macro (maybeShowDeleteUserWarning $userReference $right)
   #countPagesLastModifiedBy($userReference)
   #if ($pageCount &gt; 0)
-    &lt;div class="box errormessage xform"&gt;
+    {{/html}}
+    
+    {{error cssClass="xform"}}
```

Comment:
Why do we need this xform here? for style only? My understanding is that the goal of the ticket was to simplify the usages of errormessage, so it feels a bit too complex than it should here?

Reply:
> Why do we need this xform here? for style only?

Yup, I want to minimize the changes in the actual content produced in order to minimize breaking customization and other features. I forgot the exact context here, but I don't want to break the form (style and functionality) just because I removed a class. Having this class is supported by the XWiki macro, so I figured it was okay to refactore things like that.

> My understanding is that the goal of the ticket was to simplify the usages of errormessage

The goal of the PR is to factorize their code. This way, next time we update the message boxes, there won't be inconsistencies popping up everywhere (and this also fixes inconsistencies that popped up since we added icons with XWIKI-21452 :) ).
In order to do so, in all of the scripts here, I either use the new velocimacro, or the XWiki macro if possible.